web/testing/empty-row/empty_row-1.0.js

```diff
@@ -0,0 +1,521 @@
+KeymanWeb.KR(new Keyboard_empty_row());
+
+function Keyboard_empty_row() {
+    this.KI = "Keyboard_empty_row";
+    this.KN = "empty row";
```

Comment:
This case is more of a `Keyboard_undefined_row` - the keys array is undefined.

It may be wise to have both this case and an `Keyboard_empty_row` - where the top row's `.keys` entry is defined, but just as `[]`.

Reply:
Tweaked row 1 (see above)

Reply:
Right, I suppose the `afghan` test-resource does test the `undefined` case well enough.  :+1: